### PR TITLE
Update Overview of APIs for March 2024

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article sections for review:
* **Browser features** section: added **Hit-testing of mouse-clicks in regions** subsection.
   * https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3096#hit-testing-of-mouse-clicks-in-regions
* **Process management** section - added Win32 link **ICoreWebView2ProcessFailedEventArgs3**.
   * https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3096&tabs=win32cpp#process-management

This PR updates Overview of APIs for March 2024 promotions to Stable in Release SDK:
https://learn.microsoft.com/microsoft-edge/webview2/release-notes#10242047

Outline of headings: 
```
# Overview of WebView2 features and APIs

## Main classes: Environment, Controller, and Core

## Web/native interop
#### Host/web object sharing
#### Script execution
#### Web messaging
#### Script dialogs
#### Shared buffer

## Browser features
#### Printing
#### Cookies
#### Image capture
#### Downloads
#### Permissions
#### Context menus
#### Status bar
#### User Agent
#### Autofill
#### Audio
#### Hit-testing of mouse-clicks in regions - NEW SECTION
#### Swipe gesture navigation
#### Enable or disable the browser responding to accelerator keys (shortcut keys)
#### Fullscreen
#### PDF toolbar
#### Theming
#### Language
#### New window
#### Close window
#### Document title
#### Favicon
#### Security and privacy
###### Tracking prevention
###### SmartScreen
###### Custom crash reporting
#### Browser extensions

## Process management
#### Frame process info

## Navigate to pages and manage loaded content
#### Manage content loaded into WebView2
#### Navigation history
#### Navigation kind
#### Block unwanted navigating
#### Navigation events
#### Manage network requests in WebView2
#### Custom scheme registration
#### Client certificates
#### Server certificates
#### Launch an external URI scheme

## iframes

## Authentication

## Rendering WebView2 in non-framework apps
#### Sizing, positioning, and visibility
#### Zooming
#### Rasterization scale
#### Focus and tabbing
#### Parent window
#### Keyboard accelerators
#### Default background color

## Rendering WebView2 using Composition
#### Connecting to the visual tree
#### Forwarding input
#### Drag and drop
#### Accessibility

## User data
#### Multiple profiles
#### Delete a profile

## Performance and debugging
#### Memory usage target

## Chrome DevTools Protocol (CDP)

## See also
```

AB#49086484

AB#49361370

### CODE REVIEW

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 1]
```diff
@@ -34,6 +34,102 @@ When hosting the WebView2 control, your app has access to the following features
 This page only lists APIs that are in Release SDKs; it doesn't list Experimental APIs, or Stable APIs that are not yet available in Release SDKs.  For a comprehensive list of APIs including Experimental APIs, see [Release Notes for the WebView2 SDK](../release-notes/index.md).
 
 
+<!-- ====================================================================== -->
+## incoming
+
+<!-- ---------- -->
+* Added a new API to provide hit-testing results on the regions that a WebView2 contains.  This API is useful for visually hosted applications that want to handle mouse events on the non-client area of the WebView2 window.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:41]: What section does this API fit it?
Outline: topmost comment in PR main page:
https://github.com/MicrosoftDocs/edge-developer/pull/3096
Rendered: 
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3096#incoming

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 2]
```diff
@@ -34,6 +34,102 @@ When hosting the WebView2 control, your app has access to the following features
...
+* [COREWEBVIEW2_NON_CLIENT_REGION_KIND enum](/microsoft-edge/webview2/reference/win32/webview2-idl#corewebview2_non_client_region_kind)
+   * `COREWEBVIEW2_NON_CLIENT_REGION_KIND_CAPTION`
+   * `COREWEBVIEW2_NON_CLIENT_REGION_KIND_CLIENT`
+   * `COREWEBVIEW2_NON_CLIENT_REGION_KIND_NOWHERE`
+
+---
+
+
+<!-- ---------- -->
+* Added the `FailureSourceModulePath` property to the `ProcessFailedEventArgs` type, to specify the full path of the module that caused the crash in cases of Windows code integrity failures - that is, when a process exited with `STATUS_INVALID_IMAGE_HASH`.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:113]: What section does this Process Failed Event Args API fit in?
Outline: topmost comment in PR main page:
https://github.com/MicrosoftDocs/edge-developer/pull/3096
Rendered:
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3096#incoming
@victorhuangwq @peiche-jessica

Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md:113]: will move into section:
```
## Process management
#### Frame process info
```

Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md]: moved into tab-set in "Process management" h2 section
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3096#process-management

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 3]
```diff
@@ -34,6 +34,102 @@ When hosting the WebView2 control, your app has access to the following features
...
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2ProcessFailedEventArgs` Class:
+    * [CoreWebView2ProcessFailedEventArgs.FailureSourceModulePath Property](/dotnet/api/microsoft.web.webview2.core.corewebview2processfailedeventargs.failuresourcemodulepath)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md]: Delete member?  Class has multiple members, not each linked:
https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2processfailedeventargs --
ExitCode
FailureSourceModulePath
FrameInfosForFailedProcess
ProcessDescription
ProcessFailedKind
Reason

Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md]: Deleted the member link.

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md:41]: idea: find "mouse", "click", in outline

Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md]: Moved hit-testing section to:
```
## Browser features
...
#### Audio
#### Hit-testing of mouse-clicks in regions
#### Swipe gesture navigation
...
```
@victorhuangwq @peiche-jessica

File: microsoft-edge/webview2/release-notes/index.md [hunk 4]
```diff
@@ -122,9 +119,6 @@ The following APIs have been promoted to Stable and are now included in this Rel
    * `Client`
    * `Nowhere`
 
-* `CoreWebView2ProcessFailedEventArgs` Class:
-   * [CoreWebView2ProcessFailedEventArgs.FailureSourceModulePath Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2processfailedeventargs?view=webview2-winrt-1.0.2420.47&preserve-view=true#failuresourcemodulepath)
```
Comment (victorhuangwq) [inline microsoft-edge/webview2/release-notes/index.md:126]: May I ask why is this removed?

Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes/index.md:126]: There's an error in March relnotes for Release, Promotions to stable: 
https://learn.microsoft.com/microsoft-edge/webview2/release-notes/#promotions

So, in .NET tab & WinRT tab, delete this item here, which is dup'd from the subsequent Promotion entry: 
CoreWebView2ProcessFailedEventArgs.FailureSourceModulePath Property

Rendered PR: 
https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3096&tabs=dotnetcsharp#promotions -- item has been removed from first tab-set, b/c it's properly already in the 2nd tab-set.

**Review by victorhuangwq — APPROVED** [review]
LGTM
